bin/preview-deploy/aws.user-data.sh

```diff
@@ -352,10 +352,7 @@ cd ~
 # Move the API code into place, then go set it up
 mv eAPD/api/* /app/api
 cd /app/api
-npm ci --only=production
-# Build and seed the database
-NODE_ENV=development DEV_DB_HOST=localhost npm run migrate
-NODE_ENV=development DEV_DB_HOST=localhost npm run seed
+
 # pm2 wants an ecosystem file that describes the apps to run and sets any
```

Comment:
Love the comments it's such a joy to read and understand the next part completely.